packages/components/pie-icon-button/src/iconButton.scss

```diff
@@ -0,0 +1,122 @@
+:host {
+    --btn-border-radius: var(--dt-radius-rounded-e);
+
+    // The base values are set to the size default, which is for the medium button size
+    --btn-height: 48px;
+    --btn-width: var(--btn-height);
+
+    // The following values set to default background and color
+    // currently this sets the primary button styles
+    --btn-bg-color: var(--dt-color-interactive-brand);
+    --btn-icon-fill: var(--dt-color-content-interactive-primary);
+    --btn-focus: var(--dt-color-focus-outer);
+}
+
+@mixin button-interactive-states($bg-color) {
+    & .o-iconBtn:hover:not(:disabled) {
+        background-color: hsl(var(#{$bg-color}-h), var(#{$bg-color}-s), calc(var(#{$bg-color}-l) - var(--dt-color-hover-01)));
+    }
+
+    & .o-iconBtn:active:not(:disabled) {
+        background-color: hsl(var(#{$bg-color}-h), var(#{$bg-color}-s), calc(var(#{$bg-color}-l) - var(--dt-color-active-01)));
+    }
+}
+
+// Base button styles
+.o-iconBtn {
+    min-block-size: var(--btn-height);
+    aspect-ratio: 1 / 1;
+
+    @supports not (aspect-ratio: 1 / 1) {
+        min-inline-size: var(--btn-width);
+    }
+
+    border: none;
+    border-radius: var(--btn-border-radius);
+    outline: none;
+    background-color: var(--btn-bg-color);
+    cursor: pointer;
+    user-select: none;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &:focus-visible {
+        // TODO - Need to update to use the latest focus styling
```

Comment:
Forgotten TODO? Or is this for later?

Reply:
I think we need to update the design tokens first which we could do separately (needs to be done for the pie-button package too)